Sources/ConsoleKit/Command/Utilities.swift

```diff
@@ -46,3 +46,59 @@ private extension String {
         return String(repeating: " ", count: padding) + self
     }
 }
+
+extension String {
+    /// Calculate the Levenshtein distance to another
+    /// - Parameter target: Another String
+    /// - See: https://en.wikipedia.org/wiki/Levenshtein_distance
+    func levenshteinDistance(to target: String) -> Int {
```

Comment:
For organizational purposes, this method should probably be in a new file in the `Utilities` directory, or in the `CommandError.swift` file.